compiler/next/include/chpl/queries/all-global-strings.h

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2021 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* X(identifier, chapel string name) */
+/* the identifier must be a legal C++ identifier */
+
+X(align          , "align")
```

Comment:
We have been using more descriptive names for the macro when using this pattern in other places in the compiler. Here it could be `GLOBAL_STRING` rather than `X`. However I don't have a strong preference about it.

Reply:
I'm inclined to leave it as X because they are called X Macros elsewhere. Like https://en.wikipedia.org/wiki/X_Macro

Reply:
That's fine but arguably a comment at the top of our files saying this is the "X macro pattern" would be more useful. Anyway, in other places we are often needing more than one variant of the macro, and naming them `X` `Y` and `Z` is probably not helpful, so naming the macro `X` isn't really an option.

Reply:
added comment